cli/lib/stats.ts

```diff
@@ -0,0 +1,92 @@
+import { isSameDay, isSameMonth, isSameWeek } from 'date-fns';
+import fetch from 'node-fetch';
+import { readAppdata, saveAppdata } from './appdata';
+
+export enum StatsGroup {
+	STUDIO_CLI_USAGE_UNIQUE = 'studio-cli-usage-unique',
+}
```

Comment:
I'm thinking we could add this to `StatsGroup` in `src/lib/bump-stats/types.ts`. It would clarify the structure of the `lastBumpStats` object.